tools/docker/docker-installation.md

```diff
@@ -19,7 +19,7 @@ $ sudo dnf install dnf-plugins-core
 To add the `docker-ce` repository
 
 ```console
```

Comment:
That is the correct form for dnf5, however, Fedora 40 that's still supported where dnf4 is still default (at least in registry.fedoraproject.org/fedora:40 container), the original form is the correct one.

````suggestion
* on Fedora 40:
```
$ sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
```
* on Fedora 41 and newer:
```console
````

Not sure whether using a form like `* on dnf4 (Fedora 40):` is better or too much

Reply:
I figured that could be the origin of the change, but thanks for confirming! Definitely a good idea to list both.
> Not sure whether using a form like * on dnf4 (Fedora 40): is better or too much

I think stating the OS' version is enough.